UI/properties-view.cpp

```diff
@@ -217,7 +217,8 @@ QWidget *OBSPropertiesView::NewWidget(obs_property_t *prop, QWidget *widget,
 	const char *long_desc = obs_property_long_description(prop);
 
 	WidgetInfo *info = new WidgetInfo(this, prop, widget);
-	connect(widget, signal, info, SLOT(ControlChanged()));
+	if (signal != NULL)
```

Comment:
I need to check if `NULL`, because the `connect` function wait for a true signal to be put. And I rather want to skip it than putting useless log because the `connect` can't connect because we don't want to connect.

- If I didn't put the check and let `NULL` this was happening in the terminal output:
```log
QObject::connect: Cannot connect QLabel::(nullptr) to WidgetInfo::ControlChanged()
QObject::connect: Cannot connect QLabel::(nullptr) to WidgetInfo::ControlChanged()
QObject::connect: Cannot connect QLabel::(nullptr) to WidgetInfo::ControlChanged()
```
- If I didn't put it and put `nullptr` rather than `NULL` this was happening in the terminal output:
```log
QObject::connect: Cannot connect QLabel::(nullptr) to WidgetInfo::ControlChanged()
QObject::connect: Cannot connect QLabel::(nullptr) to WidgetInfo::ControlChanged()
QObject::connect: Cannot connect QLabel::(nullptr) to WidgetInfo::ControlChanged()
```
<!-- - The two following are surely very wrong ideas:
  - If I didn't put it and put `SIGNAL(NULL)` rather than `NULL` this was happening in the terminal output:
```log
QObject::connect: No such signal QLineEdit::editingFinished(const QString &) in /home/tytan652/Programming/OBS/obs-studio/UI/window-basic-settings-stream.cpp:97
QObject::connect:  (sender name:   'customServer')
QObject::connect:  (receiver name: 'OBSBasicSettings')
Attempted path: ../../data/obs-studio/locale.ini
Attempted path: ../../data/obs-studio/themes/
QObject::connect: Parentheses expected, signal QLabel::NULL in /home/tytan652/Programming/OBS/obs-studio/UI/properties-view.cpp:1421
QObject::connect: Parentheses expected, signal QLabel::NULL in /home/tytan652/Programming/OBS/obs-studio/UI/properties-view.cpp:1421
QObject::connect:  (sender name:   'warningLabel')
QObject::connect: Parentheses expected, signal QLabel::NULL in /home/tytan652/Programming/OBS/obs-studio/UI/properties-view.cpp:1421
```
  - If I didn't put it and put `SIGNAL(nullptr)` rather than `NULL` this was happening in the terminal output:
```log
QObject::connect: No such signal QLineEdit::editingFinished(const QString &) in /home/tytan652/Programming/OBS/obs-studio/UI/window-basic-settings-stream.cpp:97
QObject::connect:  (sender name:   'customServer')
QObject::connect:  (receiver name: 'OBSBasicSettings')
Attempted path: ../../data/obs-studio/locale.ini
Attempted path: ../../data/obs-studio/themes/
QObject::connect: Parentheses expected, signal QLabel::nullptr in /home/tytan652/Programming/OBS/obs-studio/UI/properties-view.cpp:1421
QObject::connect: Parentheses expected, signal QLabel::nullptr in /home/tytan652/Programming/OBS/obs-studio/UI/properties-view.cpp:1421
QObject::connect:  (sender name:   'warningLabel')
QObject::connect: Parentheses expected, signal QLabel::nullptr in /home/tytan652/Programming/OBS/obs-studio/UI/properties-view.cpp:1421
``` -->

Or there is another solution and I don't know about it.